### PR TITLE
Fix create_pods fixture

### PR DESCRIPTION
**Pull request by ebenahar**
Signed-off-by: Elad Ben Aharon <ebenahar@redhat.com>

### CODE REVIEW

File: tests/fixtures.py [hunk 1]
```diff
@@ -260,7 +260,12 @@ def finalizer():
 
     request.addfinalizer(finalizer)
 
-    class_instance.pod_objs = helpers.create_pods(pvc_objs_list=class_instance.pvc_objs)
+    class_instance.pod_objs = [
+            helpers.create_pod(
+                interface_type=class_instance.interface, pvc_name=pvc_obj.name,
+                do_reload=False, namespace=class_instance.namespace
+            ) for pvc_obj in class_instance.pvc_objs
+        ]
```
Comment (prsurve) [inline tests/fixtures.py]: indentation missing

**Comment by ebenahar** [comment]
[test_logs.zip](https://github.com/red-hat-storage/ocs-ci/files/3516945/test_logs.zip)

**Comment by vasukulkarni** [comment]
@ebenahar needs all tier1 testing?

**Comment by ebenahar** [comment]
@vasukulkarni I don't think this requires entire tier1 execution as this affects only a handful of test cases

**Pull request merged**